### PR TITLE
Fix editorial issues in Relative DID URLs section.

### PR DESCRIPTION
**Pull request by msporny**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/629.html" title="Last updated on Feb 16, 2021, 2:44 PM UTC (47cddc3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/629/1dda7f5...47cddc3.html" title="Last updated on Feb 16, 2021, 2:44 PM UTC (47cddc3)">Diff</a>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by dhh1128** [comment]
I am not on a device where I can login to github right now, but I believe I
can respond to an email with input. So this is a message saying that I,
Daniel Hardman, approve the pool request and vote that it get merged.

**Comment by msporny** [comment]
> I am not on a device where I can login to github right now, but I believe I can respond to an email with input. So this is a message saying that I, Daniel Hardman, approve the pool request and vote that it get merged.

PR approval via email... love it. Thank you @dhh1128, noted.

File: index.html [hunk 1]
```diff
@@ -1068,31 +1068,31 @@ <h2>DID Parameters</h2>
         <h2>Relative DID URLs</h2>
 
         <p>
-A relative <a>DID URL</a> is any URL value in a <a>DID document</a> that does not start
-with <code>did:&lt;method-name>:&lt;method-specific-id></code>. More
+A relative <a>DID URL</a> is any URL value in a <a>DID document</a> that does
+not start with <code>did:&lt;method-name>:&lt;method-specific-id></code>. More
 specifically, it is any URL value that does not start with the ABNF defined  in
```
Comment (dlongley) [inline index.html:1073]:
```suggestion
specifically, it is any URL value that does not start with the ABNF defined in
```

File: index.html [hunk 2]
```diff
@@ -1068,31 +1068,31 @@ <h2>DID Parameters</h2>
...
-Section <a href="#did-syntax"></a>. The contents of the URL typically
-refers to a <a>resource</a> in the same <a>DID document</a>. Relative <a>DID URLs</a>
-MAY contain relative path components, query parameters, and fragment
-identifiers.
+<a href="#did-syntax"></a>. The contents of the URL typically refers to
+a <a>resource</a> in the same <a>DID document</a>. Relative <a>DID URLs</a> MAY
```
Comment (dlongley) [inline index.html:1075]:
```suggestion
<a href="#did-syntax"></a>. The URL is expected to reference
a <a>resource</a> in the same <a>DID document</a>. Relative <a>DID URLs</a> MAY
```

File: index.html [hunk 3]
```diff
@@ -1068,31 +1068,31 @@ <h2>DID Parameters</h2>
...
+<strong>path</strong>, <strong>query</strong>, and <strong>fragment</strong>
+values are those defined in <a href="#path"></a>, <a href="#query"></a>, and <a
+href="#fragment"></a>, respectively.
         </p>
 
         <p>
-Relative <a>DID URLs</a> are often used to identify <a>verification methods</a> and
-<a>services</a> in a <a>DID Document</a> without having to use absolute URLs, which
-tend to be more verbose than necessary.
+Relative <a>DID URLs</a> are often used to identify <a>verification methods</a>
```
Comment (dlongley) [inline index.html:1092]:
```suggestion
Relative <a>DID URLs</a> are often used to reference <a>verification methods</a>
```

**Review by dlongley — APPROVED** [review]

**Review by rhiaro — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**